contributors/guide/contributor-cheatsheet/README-es.md

```diff
@@ -0,0 +1,396 @@
+# Cheat Sheet Hoja de Trucos para Contribuyentes de Kubernetes
+
+Una lista de recursos comunes al contribuir a Kubernetes, consejos, trucos y mejores prácticas utilizadas dentro del proyecto Kubernetes. 
+Es una "TL;DR" o referencia rápida de información útil para mejorar tu experiencia de colaboración en GitHub.
+
+**Tabla de Contenidos**
+
+- [Recursos Útiles](#recursos-útiles)
+  - [Empezando](#empezando)
+  - [SIGs y Otros Grupos](#sigs-y-otros-grupos)
+  - [Comunidad](#comunidad)
+  - [Flujo de Trabajo](#flujo-de-trabajo)
+  - [Pruebas](#pruebas)
+  - [Alias de Correo Importantes](#alias-de-correo-importantes)
+  - [Otros Enlaces Útiles](#otros-enlaces-útiles)
+- [Comunicación Efectiva en GitHub](#comunicación-efectiva-en-github)
+  - [Cómo Ser Excelente Entre Sí](#cómo-ser-excelente-entre-sí)
+    - [Ejemplos de Buena/Mala Comunicación](#ejemplos-de-buenamala-comunicación)
+- [Presentar una Contribución](#presentar-una-contribución)
+  - [Firmar el CLA](#firmar-el-cla)
+  - [Abrir y Responder a Issues](#abrir-y-responder-a-issues)
+    - [Crear un Issue](#crear-un-issue)
+    - [Responder a un Issue](#responder-a-un-issue)
+  - [Abrir un Pull Request](#abrir-un-pull-request)
+    - [Crear un Pull Request](#crear-un-pull-request)
+    - [Ejemplo de Descripción de un Pull Request](#ejemplo-de-descripción-de-pull-request)
+    - [Solucionar Problemas de un Pull Request](#solucionar-problemas-de-un-pull-request)
+  - [Labels](#labels)
+- [Trabajar Localmente](#trabajar-localmente)
+  - [Estrategia de Branch](#estrategia-de-branch)
+    - [Agregar Upstream](#agregar-upstream)
+    - [Mantener tu Fork Sincronizado](#mantener-tu-fork-sincronizado)
+    - [Combinar Commits (Squashing Commits)](#squashing-commits)
+   
+---
+## Recursos Útiles
+
+### Empezando
+
+- [Curso para Colaboradores] - **NUEVO** - El curso de E-Learning para colaboradores de Kubernetes!
+- [Guía para Colaboradores] - Guía sobre cómo comenzar a contribuir al Proyecto
+  Kubernetes.
+- [Guía del Desarrollador] - Guía para contribuir código directamente al Proyecto
+  Kubernetes.
+- [Guía de Seguridad y Divulgación] - Guía para informar vulnerabilidades
+  y el proceso de liberación de seguridad.
```

Comment:
Esto no es incluyendo en el sección de links. https://github.com/kubernetes/community/blob/46e953e913aa8b11d4ea85d0992e5e9e18ff637f/contributors/guide/contributing.md#security

Reply:
The link is at the bottom of the page as a reference.